README.md

```diff
@@ -12,39 +12,39 @@ If you are using IntelliJ IDEA to edit morphir-jvm's Scala code, you can create
 IntelliJ project files via:
 
 ```bash
-./mill mill.scalalib.GenIdea/idea
+.\mill mill.scalalib.GenIdea/idea
```

Comment:
The backslash only applies to Windows and "bash" suggests that this was intended for Unix. If you want to make this change you should have two sections, one for Unix with the existing and another for Windows with the backslash.